File: C/ex31.c
//Exibir os vinte primeiros valores da série de Bergamaschi. A série: 1, 1, 1, 3, 5, 9, 17, ...


#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    
    // Declaração de variáveis
  int a, b, c, r;
 a=1;
 b=1;
 c=1;
   getch ();
  printf("\n A serie de Bergamaschi ate 20",r); 
  printf("%d\n%d\n%d\n", a,b,c);

  for(int i = 0; i < 18; i++){
   r=a+b+c;
   a=b;
   b=c;
   c=r;

   printf("%i\n",r);
  }


    return 0;
}

File: Python/ex31.py
#Exibir os vinte primeiros valores da série de Bergamaschi. A série: 1, 1, 1, 3, 5, 9, 17, ...

n1=1
n2=1
n3=1
  
print("\nOs primeiros 20 numeros da sequencia de Bergamaschi sao: ")
print("\n", n1, "\n", n2, "\n", n3)

for i in range (3, 20, 1):
    soma=n1+n2+n3   
    print("",soma)    
    n1=n2
    n2=n3
    n3=soma